src/Websites/Platform/src/Bit.Websites.Platform.Client/Shared/Header.razor.scss

```
@@ -216,23 +321,25 @@
+    transform: translateY(rem2(-8px));
     border: 1px solid $bit-color-border-secondary;
     background-color: $bit-color-background-secondary;
-
-    @include sm {
-        padding: 0;
-        overflow: auto;
-        max-height: 25rem;
-        flex-flow: column nowrap;
-    }
+    box-shadow: 0 rem2(8px) rem2(24px) rgba(0, 0, 0, 0.12);
+    // Visibility flips at the END of the fade-out (the 180ms delay) and instantly on fade-in.
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)